ingestion-beam/src/test/resources/testdata/basic-messages-valid-null-attributes.ndjson

```diff
@@ -0,0 +1,4 @@
+{"attributeMap":null,"payload":"dGVzdA=="}
+{"attributeMap":null,"payload":"dGVzdA=="}
+{"attributeMap":null,"payload":"dGVzdA=="}
+{"attributeMap":null,"payload":""}
```

Comment:
i think we should keep these in the same order as the source, even though the tests don't check order, so that it's more obvious to humans what is responsible for what.

Reply:
That's legit. I just pushed a change that puts the null payload as the third of 4 messages consistently and also removes the `mixed-validity` file in favor of a glob on the valid and invalid files, so we have one less file to keep in sync.